sihl/src/web_migration.ml

```diff
@@ -0,0 +1,62 @@
+let log_src = Logs.Src.create "sihl.middleware.migration"
+
+module Logs = (val Logs.src_log log_src : Logs.LOG)
+
+exception Exception
+
+let template migrations =
+  let migrations = migrations |> List.map fst |> String.concat ", " in
+  Format.asprintf
+    {|
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Pending migrations</title>
+  <style>
+    %s
+  </style>
+</head>
+
+<body class="antialiased">
```

Comment:
Where are all these class names coming from?

Reply:
From the imported hardcoded error style.